pkg/client/client_test.go

```diff
@@ -359,14 +301,17 @@ func TestClient_ReadDocument(t *testing.T) {
 
 	client := New("http://" + srvAddr + "/encrypted-data-vaults")
 
-	validConfig := getTestValidDataVaultConfiguration(false)
-	_, err := client.CreateDataVault(&validConfig)
+	validConfig := getTestValidDataVaultConfiguration()
+	vaultLocationURL, err := client.CreateDataVault(&validConfig)
 	require.NoError(t, err)
 
-	_, err = client.CreateDocument(testVaultID, getTestValidEncryptedDocument())
+	vaultLocationSplitUp := strings.Split(vaultLocationURL, "/")
+	vaultID := vaultLocationSplitUp[len(vaultLocationSplitUp)-1]
```

Comment:
I see this URL splitting stuff done in a couple places in these tests - you could extract that out to a function

Reply:
fixed